tool/RELEASE_INSTRUCTIONS.md

```diff
@@ -28,8 +28,22 @@ Make sure:
 
 > If you need to install the [Github CLI](https://cli.github.com/manual/installation) you can run: `brew install gh`
 
-> Ensure the `devtools_tool` executable has been globally activated:
-> `flutter pub global activate --source path tool`
+- Ensure that you have access to `devtools_tool`
+  - For MacOS Users
+    - Ensure the `devtools_tool` executable is in your path:
+      - add the following to your `~/.bashrc` file.
+        - `export PATH=$PATH:<DEVTOOLS_DIR>/tool/bin`
+          > [!NOTE]  
+          > Replace `<DEVTOOLS_DIR>` with the local path to your DevTools
+          > repo path.
+  - For Windows Users
+    - Ensure the `devtools_tool` executable has been globally activated:
+      - `flutter pub global activate --source path tool`
+        > [!WARNING]  
+        > Always do this activation step before running
+        > `devtools_tool`. If there have been changes to the tool since the
+        > last time you have run the activate, then it needs to be rerun to
+        > pick up those changes.
```

Comment:
@DanTup is there a better way to put `devtools_tool` on the global path for windows?

Reply:
I don't think so - I think `global activate` is the right way, though you need to then ensure the Pub `bin` path is in your `PATH`. Personally I tend to not bother adding it to `PATH` though and just use `dart pub global run foo` for things like this (and `npm`) because I'm always paranoid about installing something that might have a `ls` executable or something 😄

Reply:
Although - maybe I misunderstood the question.. I'm not sure why the macOS/Windows instructions are so different here - shouldn't they both be approximately the same (ensure the Pub `bin` folder is on `PATH` and then globally activate)?

Reply:
This PR actually moves away from using pub global activate on mac and linux. Since the `devtools_tool` executable is from the tool/ package, which is activated from source, this can get out of date when working on the tool package. So to prevent having a stale `devtools_tool` exe, Dan added a script `devtools_tool` in the same directory as `devtools_tool.dart`, and then put the `devtools/tool/bin` directory on the global path so that it can be accessed via  `devtools_tool ...`. 

So the question could be, is there a way to do something similar on windows so that whenever you call `devtools_tool ...` you can be sure that you are not getting a stale `devtools_tool` exe

Reply:
Ooooh, I see!

In that case, you could do the same for Windows by having a `devtools_tool.bat` that does the same. I suspect something like (not currently at my PC so can't easily test):

```
dart run "%~dp0\devtools_tool.dart" %*
```

Then the rest would be the same (add this folder to `PATH`).

If this makes sense, I can send another PR to add/update this when back at PC tomorrow.

Reply:
I guess this isn't too different to what Flutter is doing with [dart.bat](https://github.com/flutter/flutter/blob/master/bin/dart.bat).

Reply:
@DanTup yeah if you want to push up a fix for the window side tomorrow after this lands, that sgtm. Thanks!

Reply:
Since this hasn't landed yet I pointed my PR at Dan's branch, but I can re-point it here if we want to merge this first:

https://github.com/CoderDake/devtools/pull/2